perses/rjmc/topology_proposal.py

```diff
@@ -1401,6 +1401,34 @@ class PolymerProposalEngine(ProposalEngine):
     This base class is not meant to be invoked directly.
     """
 
+    _aminos = ['ALA', 'ARG', 'ASN', 'ASP', 'CYS', 'GLN', 'GLU', 'GLY', 'HIS', 'ILE', 'LEU', 'LYS', 'MET', 'PHE',
```

Comment:
Was this accidentally swept in from another PR?

Reply:
I think this line (and the lines below it) were moved here so that we could identify which amino acids are positive vs negative. This distinction is necessary because we determine the `charge_diff` by subtracting `charge_of_old_aa` - `charge_of_new_aa`